src/Sylius/Behat/Context/Api/Admin/ManagingProductOptionsContext.php

```diff
@@ -119,6 +141,15 @@ public function theProductOptionShouldAppearInTheRegistry(ProductOptionInterface
         Assert::true($this->client->hasItemWithValue('name', $productOption->getName()));
     }
 
+    /**
+     * @Then /^(this product option) name should be "([^"]+)"$/
+     */
+    public function thisProductOptionNameShouldBe(ProductOptionInterface $productOption, string $name): void
+    {
+        $this->client->show('product_options', $productOption->getCode());
```

Comment:
```suggestion
        $this->client->show('product_options', $productOption->getId());
```
?

Reply:
We defined the code as identifier of currency (check [here](https://github.com/Sylius/Sylius/blob/api/src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Currency.xml#L20)), so I've done the same with product options. It's another decision we need to take. We use codes in Sylius (ui routes), but, on the other hand, it's not reliable as it's always just and autogenerated integer ⚖️ I'm for using codes, but cc @pamil @lchrusciel

Reply:
okey I was confused because in `apiClient` we have `public function buildUpdateRequest(string $resource, string $id): void`  so in this functions we should have `$code` instead `$id` if i understand it correctly 💃

Reply:
To be honest... I'm not sure 😄 For us, in Sylius, it's a code indeed, but for ApiPlatform it's an identifier (even if you take a look at the URL there is `{id}`, not `{code}`). So it still makes sense to me to keep it as it is 🚀 But I'm not forcing it if it's confusing for some more people as well 🖖

Reply:
Maybe we should use an IRI as the unique identifier in API Platform (`ApiPlatform\Core\Api\IriConverterInterface`)?